### PR TITLE
Minor updates (mostly taus, isolation in delphes, simpler histogramer for delphes)

### PR DESCRIPTION
**Pull request by mcepeda**

### CODE REVIEW

**Comment by jkiesele** [comment]
All looks good, I just have one comment (see code comments)

File: delphesInterface/ntupler/src/ntupler.cpp [hunk 1]
```diff
@@ -138,18 +180,18 @@ void ntupler::analyze(size_t childid /* this info can be used for printouts */){
...
+		for(size_t i=0;i<selectedMuons.size();i++){
 			if(ev_.ntm>=MiniEvent_t::maxpart)break;
-			ev_.tm_pt    [ev_.ntm] =muontight.at(i)->PT;
-			ev_.tm_eta   [ev_.ntm]=muontight.at(i)->Eta;
-			ev_.tm_phi   [ev_.ntm]=muontight.at(i)->Phi;
+			ev_.tm_pt    [ev_.ntm] =selectedMuons.at(i)->PT;
+			ev_.tm_eta   [ev_.ntm]=selectedMuons.at(i)->Eta;
+			ev_.tm_phi   [ev_.ntm]=selectedMuons.at(i)->Phi;
 			ev_.tm_mass  [ev_.ntm]=0.105;
-			ev_.tm_relIso[ev_.ntm]=muontight.at(i)->IsolationVarRhoCorr/muontight.at(i)->PT;
```
Comment (mcepeda) [inline delphesInterface/ntupler/src/ntupler.cpp]: I think the isolation is already relative, to be checked

File: delphesInterface/ntupler/src/minivalidator.cpp [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+        chilidstr+=childid;
+        TFile * outfile= new TFile(getOutDir()+"/p2val_"+(TString)getLegendName()+"_"+chilidstr+".root","RECREATE");
+        TDirectory *counterdir = outfile->mkdir("weightCounter");
+        counterdir->cd();
+        TH1F * h_event_weight = new TH1F("Event_weight","Event_weight",1,0,1);
+
+        outfile->cd();
+        TDirectory *muondir = outfile->mkdir("muonTight");
+        muondir->cd();
+        TH1F * h_muontight_all_pt = new TH1F("h_muontight_all_pt","Muon PT",200,0,200);
```
[the remark this answers is not in the document]
Comment (mcepeda) [reply delphesInterface/ntupler/src/minivalidator.cpp]: I would like to keep by now the directory structure - can we merge like this for now while we develop it? (in principle the validation should be run similarly on top of the ntuples, so I wanted to keep that part simple for the time being)

[the remark this answers is not in the document]
Comment (jkiesele) [reply delphesInterface/ntupler/src/minivalidator.cpp]: +1

**Pull request merged**